www/404.html

```diff
@@ -32,9 +32,9 @@
   <link rel="shortcut icon" type="image/x-icon" href="favico.ico">
   <meta name="csrf-param" content="authenticity_token">
 <meta name="csrf-token" content="2B6Cdu3FaYKkvcjpi0RUtuJbyUTRtHfNp7OEb0NajaKpHYB7Hk32FEBvu6wwVji1VFQRrv0pG368Rw3cszejUQ==">
```

Comment:
I realise this is unrelated to your change, but I'll note that a hardcoded CSRF token isn't as useful as it might appear at first glance.

@m-richo - I'm guessing you pulled that when you converted the old site to be static?

Unless we're processing these elsewhere, we should probably delete (possibly in a separate PR).

Reply:
Yeah it can be deleted

Reply:
yeah, i just scraped the live site to static files and it must have added the CSRF token.  Let's remove after this is merged.